src/openssl.c

```diff
@@ -10292,8 +10331,12 @@ static const auxL_Reg sx_methods[] = {
 #endif
 #if HAVE_SSL_CTX_GET0_CHAIN_CERTS
 	{ "getCertificateChain", &sx_getCertificateChain },
+#endif
+#if HAVE_USE_CERTIFICATE_CHAIN_FILE
+	{"setCertificateChainFromFile", &sx_useCertificateChainFile},
 #endif
 	{ "setPrivateKey",    &sx_setPrivateKey },
+	{ "setPrivateKeyFromFile", &sx_usePrivateKeyFile},
```

Comment:
please realign the section

Reply:
Attempted to realign the reg declarations in keeping with the current style.